converter/bodies.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+from abc import ABC
+
+
+@dataclass(frozen=True)
+class Body(ABC):
```

Comment:
I'd use a more generic name of SolidFigure, see
https://en.wikipedia.org/wiki/Solid_geometry#Solid_figures

Body is SHIELD-HIT12A specific, other codes like TOPAS do not have notion of bodies.

Reply:
As suggested i went with 'SolidFigure" for `Body` class and renamed the other classes accordingly.

Reply:
ok